### PR TITLE
Sites

### PR DESCRIPTION
**Pull request by Caleb-Nieto**
### Feature / Bug Description
Sites in a table

### Solution
Add the necessary components

### Tickets
[WMA-60](https://cdentalcaregroup.atlassian.net/browse/WMA-60)

### Screenshots / Screencasts
table
Captura de pantalla 2024-06-05 114401

[WMA-60]: https://cdentalcaregroup.atlassian.net/browse/WMA-60?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/company/components/CompanyCard.tsx [hunk 1]
```diff
@@ -34,7 +34,8 @@ const CompanyCard = ({ data }: CompanyCardProps) => {
     {
       key: "1",
       label: (
-        <ViewPrioritiesButton id={data.id} companyName={data.name}/>
+        //<ViewPrioritiesButton id={data.id} companyName={data.name}/>
```
Comment (imadiaz) [inline src/pages/company/components/CompanyCard.tsx:37]: can u remove this comment?

File: src/pages/site/components/SiteTable.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    isLoading: boolean;
+  }
+
+const SiteTable = ({data, isLoading}: TableProps) => {
+    const contentRef = useRef<HTMLDivElement>(null);
+  const tableHeight = useTableHeight(contentRef);
+  //const dispatch = useAppDispatch();
+
+  /* const handleUpdateClick = (row: Company) => {
+    dispatch(resetRowData());
```
Comment (imadiaz) [inline src/pages/site/components/SiteTable.tsx:22]: are you going to use this?

**Review by imadiaz — APPROVED** [review]
lgtm just some comments

**Comment by Caleb-Nieto** [comment]
Hi, I will use all the commented lines

**Pull request merged**